client/src/tsconfig.json

```diff
@@ -3,7 +3,7 @@
 		"target": "es6",
 		"module": "commonjs",
 		"moduleResolution": "node",
-		"sourceMap": false,
+		"sourceMap": true,
```

Comment:
Not "bad", but uncommon for libraries to publish source maps. (Even if they're published, it's unlikely unless the sourcemaps are inlined that they would be picked up by your apps asset pipeline). And I doubt the vscode team would want this upstreamed.